decidim-core/app/views/pages/home/_highlighted_processes.html.erb

```diff
@@ -1,37 +1,5 @@
 <section class="wrapper-home">
-  <div class="row">
-    <h3 class="section-heading"><%= t(".active_processes") %></h3>
-    <div class="row collapse">
-      <div class="row small-up-1 smallmedium-up-2 mediumlarge-up-3
-                  large-up-4 card-grid">
-        <% highlighted_participatory_processes.each do |process| %>
-          <div class="column">
-            <article class="card card--process card--mini">
-              <%= link_to decidim_participatory_processes.participatory_process_path(process), class: "card__link" do %>
-                <div class="card__image-top"
-                     style="background-image:url(<%= process.hero_image.url %>)"></div>
-              <% end %>
-              <div class="card__content">
-                <%= link_to decidim_participatory_processes.participatory_process_path(process), class: "card__link" do %>
-                  <h4 class="card__title"><%= translated_attribute process.title %></h4>
-                <% end %>
-                <% if process.active_step %>
-                  <span class="card--process__small">
-                    <%= t(".active_step") %>
-                    <strong><%= translated_attribute process.active_step.title %></strong>
-                  </span>
-                <% end %>
-              </div>
-            </article>
-          </div>
-        <% end %>
-      </div>
-    </div>
-  </div>
-  <div class="row">
-    <div class="columns small-centered small-12
-                smallmedium-8 medium-6 large-4">
-      <%= link_to t(".see_all_processes"), decidim_participatory_processes.participatory_processes_path, class: "button expanded hollow button--sc home-section__cta" %>
-    </div>
-  </div>
+  <% Decidim.view_hooks.get(:highlighted_elements).each do |hook| %>
```

Comment:
Should we prepend the hook with the engine name?

Reply:
Wouldn't do that. If another engine wants to expose view hooks, can be scoped to its constant: `Decidim::Proposals.view_hooks`. In fact, I'd move all this inside `Decidim::Core.view_hooks`.